server/src/main/java/org/elasticsearch/index/mapper/BlockStoredFieldsReader.java

```diff
@@ -63,6 +63,33 @@ public final SortedSetDocValues ordinals(LeafReaderContext context) {
         }
     }
 
+    public static class BytesFromMixedStringsBytesRefBlockLoader extends StoredFieldsBlockLoader {
```

Comment:
If this can be moved to mapper-extras module then we can make this package protected and control its usage. Given that normally this class shouldn't be used. It just now a workaround for a bwc bug.